scripts/solve_network.py

```diff
@@ -101,8 +101,9 @@ def prepare_network(n, solve_opts):
 
     if solve_opts.get('load_shedding'):
         n.add("Carrier", "Load")
-        n.madd("Generator", n.buses.index, " load",
-               bus=n.buses.index,
+        buses_i = n.buses.query('carrier not in @n.stores.carrier.unique()').index
```

Comment:
Yes, does what we want, but may be better to formulate positively (in case at some point we ever want to add a store to an AC bus).
```suggestion
        buses_i = n.buses.loc[n.buses.carrier == 'AC'].index
```

Reply:
common ground ?
```suggestion
        buses_i = n.buses.query("carrier == 'AC'").index
```

Reply:
ok